.github/workflows/main.yaml

```diff
@@ -0,0 +1,45 @@
+name: build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch: # allows you to trigger manually
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
This looks wrong.  It is checking for the string `skip ci` in the commit message, but the commit message given below includes `ci skip`.

Reply:
My bad... Good catch... This is a bug. I'll fix it

Reply:
See #115